tests/libtest.sh

```diff
@@ -652,6 +652,16 @@ which_gpg () {
     echo ${gpg}
 }
 
+has_libsodium () {
+    local ret
+    ${CMD_PREFIX} ostree --version > version.txt
+    grep -q -e '- libsodium' version.txt
```

Comment:
You could save a temporary file by simply piping the output of ostree --version to the grep. (and thus remove the rm line)

Reply:
yeah. I thought so initially. Unfortunately sometimes pipe fail for some reason, so it return negative value. In tests there function is used twice it leads to incorrect detection of tests which should be run.
I fall into that behavior while reworking `has_gpgme` function above.

Reply:
what is ${CMD_PREFIX}  ? `sudo`?

Reply:
nope. defined in lib file for shell.